### ISSUE TITLE
[Package Request] - Python 3.13

### ISSUE THREAD

**Issue by rgoltz** [amazonlinux/amazon-linux-2023#811]
**What package is missing from Amazon Linux 2023? Please describe and include package name.**
Please add Python 3.13 to Amazon Linux 2023 as package via AWS-managed repo.
https://blog.python.org/2024/10/python-3130-final-released.html (released Monday, October 7, 2024)

**Is this an update to existing package or new package request?**
Update to existing package for Python on AL2023 (would be nice to have Python 3.12 and Python 3.13 available as a transition)

**Is this package available in Amazon Linux 2? If it is available via external sources such as EPEL, please specify.**
AL2 getting close to EOL. I'm thinking it's not relevant for AL2 anymore.

**Any additional information you'd like to include. (use-cases, etc)**
This issue should be (at least) done, once [Lambda](https://github.com/aws/aws-lambda-base-images/issues/) + [CodeBuild](https://github.com/aws/aws-codebuild-docker-images/issues/) supports Python 3.13. This enable us to use different AWS services smoothly together.

While opening the issue today, the support for Python 3.12 is not covered in AL2023 yet as well: https://github.com/amazonlinux/amazon-linux-2023/issues/483. This should be done first.

**Comment by obi-jerome** [comment]
I switched to uv to get the Python Version I want...
https://github.com/astral-sh/uv

**Comment by ozbenh** [comment]
We released python3.12 recently

**Comment by eric-spitler** [comment]
I would expect 3.13 to take even longer for availability than 3.12 since it has more options for the build with regards to the GIL changes. Unless the delay for 3.12 was due to simply not having a build pathway defined, but even then it will still require more consideration for 3.13.

**Comment by matejsp** [comment]
I agree, we should have 3.13 in repos by now. Free thread alternative builds would be nice, but for now just normal 3.13.

**Comment by ozbenh** [comment]
It's being worked on as we speak

**Comment by mwebber** [comment]
Python 3.13 was added a while back - see https://docs.aws.amazon.com/linux/al2023/release-notes/relnotes-2023.8.20250721.html
@ozbenh I think you can close this issue.

**Comment by ozbenh** [comment]
Thanks,  closing.
